config/initializers/spree_comments.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+Rails.application.config.after_initialize do
+  Spree::Admin::MainMenu::DefaultConfigurationBuilder.prepend Spree::Admin::MainMenu::DefaultConfigurationBuilderDecorator
+  Rails.application.config.spree_backend.main_menu = Spree::Admin::MainMenu::DefaultConfigurationBuilder.new.build
```

Comment:
A co jak inny gem też będzie nadpisywał main_menu w adminie?

Reply:
@pkatek this time it will only add functionality, but reuse the original builder, so if another gem does the same, it will work. The way it is done in the SEO gem could cause the issues you are talking about because it uses a custom class. I think we should change the one in the SEO gem to function like this rather than change this one.

Reply:
@pkatek it could cause issues if another gem wants to change the settings submenu itself, but this is currently the only option because deface does not work on generated ERB so it is not able to identify where to add the menu item. The only other option left is to add the item via JavaScript after page load that is not the most elegant option.

Reply:
@jibranusman95  w takim razie napraw to w gemie seo abym mógł też go zmergować.